angular.json

```diff
@@ -33,7 +33,7 @@
               "src/app/style/main.scss"
             ],
             "scripts": [
-              "node_modules/netlify-identity-widget/build/netlify-identity-widget.js"
+              "node_modules/netlify-identity-widget/releases/v1/netlify-identity-widget.js"
```

Comment:
```suggestion
              "node_modules/netlify-identity-widget/releases/v1-beta/netlify-identity-widget.js"
```

Reply:
The language feature is in the beta version of our repo.
You can check if the widget is in french by clicking on "Je suis une société de services".